docs/api/map.rst.inc

```diff
@@ -15,6 +15,7 @@
 * :doc:`/api/product_attributes`
 * :doc:`/api/product_options`
 * :doc:`/api/product_variants`
+* :doc:`/api/product_reviews`
```

Comment:
I guess that `product_reviews` should be before `product_variants` to keep alphabetical order